aospy/data_loader.py

```diff
@@ -17,8 +17,15 @@
 def rename_grid_attrs(data):
     """Rename grid attributes to be consistent with aospy conventions.
 
-    This function does not compare to Model coordinates or
-    add missing coordinates from Model objects.
+    Search all of the dataset's coords and dims looking for matches to known
+    grid attribute names; any that are found subsequently get renamed to the
+    aospy name as specified in aospy.internal_names.GRID_ATTRS.
```

Comment:
What are your thoughts on formatting?  Should we use double back-ticks on aospy.internal_names.GRID_ATTRS?